### PR TITLE
Feature/mr worldwide

### PR DESCRIPTION
**Pull request by MichaelParadis**
Handles:
 - Getting remote user pages
 - Getting their remote user public posts
 - Sending public posts for local authors
 - Friend requests to our own nodes (others are weird)
 - Follow and unfollow remote author sometimes

### CODE REVIEW

File: dispersal/settings.py [hunk 1]
```diff
@@ -158,13 +150,15 @@
...
     'DEFAULT_AUTHENTICATION_CLASSES': (
-        # 'rest_framework.authentication.BasicAuthentication',
+        'rest_framework.authentication.BasicAuthentication',
         'rest_framework.authentication.SessionAuthentication',
     ),
     'DEFAULT_PAGINATION_CLASS': 'rest_framework.pagination.PageNumberPagination',
     'PAGE_SIZE': 50
 }
 
+SITE_URL = os.environ.get('SITE_URL', 'http://127.0.0.1:8000/')
```
Comment (npwhite) [inline dispersal/settings.py:160]: does this need changed for the heroku deployment

**Review by FordMcLeod — APPROVED** [review]
"We about to climb, wild, 'cause it's about to go down" -pitbull

File: dispersal/settings.py (hunk 1 above)
Comment (MichaelParadis) [reply dispersal/settings.py:160]: Heroku has a environment variable set

Comment (FordMcLeod) [reply dispersal/settings.py:160]: 127 etc.. is the default value if the request to get environ variable fails :) aka it'll be good

File: posts/static/makepost/posts.js [hunk 2]
```diff
@@ -88,21 +88,20 @@ function submitForm() {
...
     let visibility = document.getElementById('visibility').value;
     let unlisted = document.getElementById('unlisted').checked;
-    let csrf = document.getElementsByName('csrfmiddlewaretoken')[0].value;
     let visibleToArray = [];
-
-    if (visibility === "PRIVATE") {
-        visibleTo = document.getElementsByClassName('appendedUrl');
+    if (visibility === 'PRIVATE') {
+        let visibleToArray = [];
+        let visibleTo = document.getElementsByName('visibleTo[]');
```
Comment (tlaz4) [inline posts/static/makepost/posts.js:101]: This wont submit visibleTo urls on submission of post, name visibleTo[] is no longer used.

**Pull request merged**